client/src/components/ElementEditor/Element.js

```diff
@@ -1,33 +1,52 @@
 import React, { PureComponent } from 'react';
 import { elementType } from 'types/elementType';
+import { inject } from 'lib/Injector';
 
 /**
  * The Element component used in the context of an ElementEditor shows the summary
  * of an element's details when used in the CMS, including ID, Title and Summary.
  */
 class Element extends PureComponent {
   render() {
-    const { element: { ID, Title, Summary } } = this.props;
+    const { element: { ID, Title, Summary, Type, IconClass },
+      HeaderComponent,
+      ContentComponent } = this.props;
 
     if (!ID) {
       return null;
     }
 
     return (
       <div className="element-editor__element">
-        <p>#{ID}: {Title}</p>
-        <p>{Summary}</p>
+        <HeaderComponent
+          id={ID}
+          title={Title}
+          elementType={Type}
+          fontIcon={IconClass}
+        />
+        <ContentComponent
+          summary={Summary}
+        />
       </div>
     );
   }
 }
 
 Element.propTypes = {
   element: elementType,
+
 };
 
 Element.defaultProps = {
   element: null,
 };
 
-export default Element;
+export { Element as Component };
+
+export default inject(
+  ['ElementHeader', 'ElementContent'],
+  (HeaderComponent, ContentComponent) => ({
+    HeaderComponent, ContentComponent,
+  }),
+  () => 'Element'
```

Comment:
Can we please make this `ElementEditor.ElementList.Element`? The two parents above this are using `ElementEditor` and `ElementEditor.ElementList` respectively already

Reply:
of course.